### PR TITLE
New article added - troubleshooting-cannot-upload-app-to-itunesconnec…

### PR DESCRIPTION
**Pull request by Mike4o**
…t.md

### CODE REVIEW

**Comment by nadyaA** [comment]
:+1:

File: troubleshooting/troubleshooting-cannot-upload-app-to-itunesconnect.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+## Cause
+<!-- Mandatory -->
+The `Cannot upload application` message is generic for the publish service and it is shown as the AppBuilder client is unable to read the exact error returned. There are many possible causes for the upload failure.
+
+## Solution
+<!-- Mandatory -->
+Check what the actual error returned from ITMS is in the AppBuilder In-Browser client:
+
+1. Open your project in the [AppBuilder In-Browser client](https://platform.telerik.com). 
+	> **IMPORTANT:** If you are developing your app in the AppBuilder Extension for Visual Studio, you will need to either [connect the code with the cloud](http://docs.telerik.com/platform/appbuilder/development-tools/version-control/collaborating/develop-across-clients), or [import it from archive](http://docs.telerik.com/platform/appbuilder/cordova/creating-your-app/import-from-archive) in the In-Browser client.
```
Comment (ggarabedian) [inline troubleshooting/troubleshooting-cannot-upload-app-to-itunesconnect.md:51]: Empty line between the **1. Open your...** and **IMPORTANT**. Currently, this won't render as expected.

File: troubleshooting/troubleshooting-cannot-upload-app-to-itunesconnect.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+<!-- Optional -->
+`Cannot upload application.`
+
+## Cause
+<!-- Mandatory -->
+The `Cannot upload application` message is generic for the publish service and it is shown as the AppBuilder client is unable to read the exact error returned. There are many possible causes for the upload failure.
+
+## Solution
+<!-- Mandatory -->
+Check what the actual error returned from ITMS is in the AppBuilder In-Browser client:
```
Comment (ggarabedian) [inline troubleshooting/troubleshooting-cannot-upload-app-to-itunesconnect.md:47]: You can see the actual ITMS error by publishing your app using the AppBuilder In-Browser client.

Just a suggestion.

**Comment by ggarabedian** [comment]
:+1:

**Pull request merged**